dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2888,3 +2888,108 @@ def test_parquet_pyarrow_write_empty_metadata_append(tmpdir):
         append=True,
         ignore_divisions=True,
     )
+
+
+def test_read_write_partitioned_overwrite_is_true(tmpdir):
+    # https://github.com/dask/dask/issues/6824
+
+    # Create a Dask DataFrame if size (10000, 10) with 5 partitions and write to local
+    ddf = dd.from_pandas(
+        pd.DataFrame(
+            np.random.randint(low=0, high=100, size=(10000, 10)),
+            columns=["A", "B", "C", "D", "E", "F", "G", "H", "I", "J"],
+        ),
+        npartitions=5,
+    )
+    ddf = ddf.reset_index(drop=True)
+    dd.to_parquet(ddf, tmpdir, engine="pyarrow")
```

Comment:
```suggestion
    dd.to_parquet(ddf, tmpdir, engine=engine, overwrite=True)
```
Using the `engine` fixture.
Also adding `overwrite=True` as another check that we can append to an empty directory (okay?).